distr_tracing/distr_tracing_otel/distr-tracing-otel-installing.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: ASSEMBLY
+[id="install-distributed-tracing-otel"]
+= Installing the {OTELShortName}
+include::_attributes/common-attributes.adoc[]
+:context: install-distributed-tracing-otel
+
+toc::[]
+
+:FeatureName: The {OTELOperator}
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+Installing the {OTELShortName} involves the following steps:
+
+* Installing the {OTELOperator}.
+* Creating a namespace for an OpenTelemetry Collector instance.
+* Creating an `OpenTelemetryCollector` custom resource to deploy the OpenTelemetry Collector instance.
```

Comment:
I like these task overviews you have been providing, I think they help a lot. This one is unordered, wanted to make sure that is intended :slightly_smiling_face:

Reply:
Thank you. Fixed.